content/essentials/intro-to-reactivity.mdx

```diff
@@ -138,34 +138,45 @@ This keeps the user interface and other system components synchronized with the
 #### Scalability
 
 As applications grow in complexity, managing individual signals can become challenging. 
-Stores provide a scalable solution by allowing related state to be grouped together. 
+Stores provide a simple solution that allows related states to be grouped together. 
 
 ## Tracking
 
-At its core, tracking is about registering subscriptions. 
-When a signal changes, all subscribed effects are re-run. 
-This enables the automatic propagation of changes throughout an application's components.
+At its core, tracking si fundamentally about managing subscriptions in a reactive system.
+When a signal undergoes a change, all the effects *and* memos subscribed to it are automatically re-run; this ensures the propogation of changes throughout the application's components.
 
-Some key points to understand about tracking:
+### Key concepts
 
-- Tracking subscriptions occur when a signal, or store value, is accessed within an effect. 
-In other words, when you read the value of a signal inside an effect, that effect becomes subscribed to that signal.
+* Subscriptions are initaited when a signal, or a store value, are accessed.
+This interaction leads to the accessor subscribing to the signal, establishing a dynamic connection.
+Every interaction between the signal and its accessor reinforces this subscription, guaranteeing that the associated effects are re-executed when the signal changes.
+* Both memos *and* effects have the ability to subscribe to signals.
+However, interactions with a signal outside of these won't initiate tracking, due to the absence of a subscriber, thus limiting the scope of reactivity.
+* The reactive system is built on the interplay between elements that can change (signals) and those that can be updated (effects and memos).
+The principles of tracking are not limited to these elements, however, but also extend to reactive props and store values.
+* Entities like store values and reactive props are inherently depdendent on signals.
+A "signal" stands as the only entity that operates autonomously and is reliably dependable.
```

Comment:
Confusing sentence, like what other entities are not reliably dependable? Maybe saying "objects" rather than "entities" would be simpler. And "operate autonomously" seems a way more complicated way of saying "they don't depend on anything", which I'm not if it's even worth saying 🤔 maybe one can say that signals are the leaves in the reactivity graph.